SkietbaanBE/SkietbaanBE/Controllers/FeaturesController.cs

```diff
@@ -32,5 +32,34 @@ public User Search(string username)
             }
             return null;
         }
+
+        //// PUT: api/User/Update
+        [HttpPut]
+        [ActionName("Update")]
+        public async Task<IActionResult> PutUserMember([FromBody] User user)
+        {
+            if (user.Username == null)
+            {
+                return new BadRequestObjectResult("No empty fields allowed");
+            }
+            else
```

Comment:
You don't need the else here because you have an early return

Reply:
okay made the change

Reply:
okay removed